File: src/main/java/com/damo/gestionDeStock/model/Ventes.java
package com.damo.gestionDeStock.model;

import javax.persistence.*;
import lombok.*;

import java.time.Instant;
import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
@EqualsAndHashCode(callSuper = true)
@Entity
@Table(name = "ventes")
public class Ventes extends AbstractEntity{

    @Column(name = "code")
    private String code;

    @Column(name = "datevente")
    private Instant dateVente;

    @Column(name = "commentaire")
    private String commantaire;

    @Column(name = "identreprise")
    private Integer idEntreprise;

    @OneToMany(mappedBy = "vente", fetch = FetchType.LAZY, cascade = CascadeType.ALL)
    private List<LigneVente> ligneVentes;

}

File: src/gs-api/src/models/ventes.ts
/* tslint:disable */
import { LigneVente } from './ligne-vente';
export interface Ventes {
  id?: number;
  creationDate?: number;
  lastModifiedDate?: number;
  code?: string;
  dateVente?: number;
  commantaire?: string;
  idEntreprise?: number;
  ligneVentes?: Array<LigneVente>;
  username?: boolean;
}
